internal/gc.h

```diff
@@ -119,7 +117,7 @@ void rb_gc_mark_vm_stack_values(long n, const VALUE *values);
 void *ruby_sized_xrealloc(void *ptr, size_t new_size, size_t old_size) RUBY_ATTR_RETURNS_NONNULL RUBY_ATTR_ALLOC_SIZE((2));
 void *ruby_sized_xrealloc2(void *ptr, size_t new_count, size_t element_size, size_t old_count) RUBY_ATTR_RETURNS_NONNULL RUBY_ATTR_ALLOC_SIZE((2, 3));
 void ruby_sized_xfree(void *x, size_t size);
-int rb_slot_size(void);
+void *rb_gc_rvargc_object_data(VALUE obj);
```

Comment:
Why is this function exported?
Also `rb_rvargc_payload_init` and `rb_rvargc_payload_data_ptr`.

Reply:
Thanks for catching these. I've changed it so that this function isn't exported anymore.